src/components/NavHeader/index.tsx

```diff
@@ -0,0 +1,170 @@
+import Link from 'next/link';
+import { ReactComponent as LogoIcon } from 'assets/revamp-icon/logo.svg';
+import { ReactComponent as TwitterIcon } from 'assets/revamp-icon/twitter.svg';
+import { ReactComponent as DiscordIcon } from 'assets/revamp-icon/discord.svg';
+import { ReactComponent as TelegramIcon } from 'assets/revamp-icon/telegram.svg';
+import { ReactComponent as LinkIcon } from 'assets/revamp-icon/link.svg';
+import { DownOutlined } from '@aelf-design/icons';
+import React from 'react';
+import clsx from 'clsx';
+import SideMenu from 'components/SideMenu';
+import Dropdown from 'components/Dropmenu';
+
+export type MenuItem = {
+  key: string;
+  label: React.ReactNode;
+  icon?: React.ReactNode;
+  children?: MenuItem[];
+};
+
+const items: MenuItem[] = [
+  {
+    key: 'NetworkDAO',
+    label: (
+      <Link href={'/'} className="text-[15px] font-medium text-white no-underline font-Montserrat">
+        Network DAO
+      </Link>
+    ),
+  },
+  {
```

Comment:
move the constant into another file

NavHeader/constants/index.ts

Reply:
Moved